### PR TITLE
Init CeresDB docs

### PR DESCRIPTION
**Pull request by MachaelLee**
Init CeresDB documents structure, and copy documents from [CeresDB](https://github.com/CeresDB/ceresdb)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,38 @@
-# docs
-CeresDB Documents
+![CeresDB](docs/logo/CeresDB.png)
```
Comment (jiacai2050) [inline README.md:1]: This readme should describe what this repo does, not CeresDB project itself.

You can refer those to write our own:
- https://github.com/timescale/docs
- https://github.com/pingcap/docs

Comment (MachaelLee) [reply README.md]: I had rewritten readme, please review it again.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,12 @@
-# docs
-CeresDB Documents
+# README
+
+## CeresDB Documentation
+Welcome to CeresDB documentation!
+This repository stores all the source files of [CeresDB docs website](https://docs.ceresdb.io/about.html)，the website is generated with mdbook.
```
Comment (jiacai2050) [inline README.md:5]: Links to homepage https://docs.ceresdb.io

File: README.md [hunk 3]
```diff
@@ -1,2 +1,12 @@
-# docs
-CeresDB Documents
+# README
```
Comment (jiacai2050) [inline README.md:1]: Remove this title

**Review by jiacai2050 — APPROVED** [review]
LGTM

**Pull request merged**